### DISCUSSION TITLE
Compiling Newer Version with Jyers Configurations

### DISCUSSION THREAD

**Discussion by LexusSpeed03** [Jyers/Marlin#2087] [Q&A]
Hello All,

I am wanting to update to the newest version of Marlin since seems the Jyers hasn't been updated in some time. I would love to get the configuration for E3V2-BLTouch-5x5-HS-v4.2.2-v2.0.1 and E3V2-UBL-BLTouch-10x10-HS-v4.2.2-v2.0.1 so I can compile the highspeed and other various settings.

Maybe I am missing something, but I only see one configuration file. Thank you in advance for help.

**Comment by treyswild** [comment]
Any chance of an update to Marlin 2.1.1.1?

Thank you
